### ISSUE TITLE
Not Running on Windows: Suspected Bug in Dockerfile

### ISSUE THREAD

**Issue by Chris-Maechler** [camunda/camunda-platform#480]
Somewhere after version 8.1.8 the zeebe container is constantly restarting because it cannot access files in `/usr/local/zeebe/data/` (`.topology.meta` in 8.4.0). I'm using the `.env` and `Dockerfile` from this repository. All other containers like different databases, kibana, keycloak, my own images, etc. run completely fine.

According to the official Dockerfile documentation:

> If any build steps change the data within the volume after it has been declared, those changes will be discarded.

So you would have to move `VOLUME` declarations down after you create the directories and after the `COPY` instructions.

I also read somewhere that `VOLUME` would create an empty directory if it doesn't exist yet. In this case `mkdir` would fail and because you chain with `&&` the `chown` and `chmod` would be ignored.

**Comment by dwknippers** [comment]
Yep, this issue breaks Docker on Windows for all configs.

**Comment by dwknippers** [comment]
Seems like removing older volumes, created by earlier images, solves these issues.

**Comment by aabouzaid** [comment]
@Chris-Maechler Thanks for reporting this.
Could you please provide more info about the versions? (e.g. OS, Docker daemon and CLI versions).

**Comment by Chris-Maechler** [comment]
Yes, dwknippers is right, deleting the volumes solved the problem.
